### DISCUSSION TITLE
GitHub Desktop UI suggestion/feedback

### DISCUSSION THREAD

**Discussion by flyrlabs-rch** [community/community#45161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<img width="1467" alt="Screenshot 2023-01-23 at 11 28 17 AM" src="https://user-images.githubusercontent.com/104916479/214138780-9467341f-4721-484a-9e4b-4a39b8cf09fe.png">

The name of the current branch is truncated to fit the center pane of the GitHub Desktop UI

I want to see as much of this name as possible. If the top bar was independent of the body panes, it could maximize the center section while preserving the text in the right-hand section

Thank you for your consideration

**Comment by steveward** [comment]
Thanks for the feedback  We are tracking this in https://github.com/desktop/desktop/issues/4569 if you would like to follow along for any updates.
